### ISSUE TITLE
A11y_.NET Core_WPF_ListofProducts_GroupByCategory_ScreenReader: Screen reader is unnecessarily announcing list items for sale after checking / unchecking the checkbox.

### ISSUE THREAD

**Issue by Truptik12** [dotnet/wpf#6733]
[6733.zip](https://github.com/dotnet/wpf/files/9007569/6733.zip)

### Actual:
Screen reader is unnecessarily announcing list items for sale after checking / unchecking the checkbox.

**This issue is also observed with the screen reader Narrator.**

**_This issue is observed with all the checkboxes present o this screen._**

### Expected:
Screen reader should not  unnecessarily announce list items for sale after checking / unchecking the checkbox. It should announce the results.

“[Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

**GitHubTags**:#A11yMAS;##A11yTCS;#.NET Core[WPF]-Win32-June2022;#.NET Core;#A11ySev2;#WCAG1.3.1;#Element:Checkbox;#NVDA;#Narrator;#DesktopApp;#Win11;#ColorContrast;#Benchmark;

### Environment Details:
.NET Core
WPF
Operating System: Windows 11 Enterprise 21H2

### Steps to Reproduce:

1. Launch VS 2019 Int Preview
2. Navigate to Sample Applications and right click on the DatabindingDemo and click on build.
3. Then right click on DatabindingDemo then Click on Debug-->Start New Instance. 
4. List of products screen should open.
5. Select Group by category check box.
6. Turn on the screen reader and observe the announcement.

### User Impact:
Screenreader Users will receive incorrect indications because of unnecessary content announcements.

**Comment by anjalisheel-wpf** [comment]
@Truptik12 This is working as expected, check attached video

https://user-images.githubusercontent.com/85492722/182140123-0f95bbcf-15bd-4051-9cf5-8397423299df.mov

.

**Comment by dipeshmsft** [comment]
There is a previous bug that contradicts this requirement. 
1145075 -- Screenreader does not indicate that the list of items is updated after any checkboxes checked.

This behaviour of reading the changed list items was introduced as a fix for this particular issue. Please suggest an expected behaviour in light of this issue.

**Comment by merriemcgaw** [comment]
@dipeshmsft I honestly think the other bug was incorrect and potentially should have been closed as "not a bug" but I didn't know there was any question on the specifics of the case, or why .NET 6 testing didn't show this. That said, if some status on the window changes after the user select/deselects a checkbox, the screen reader really needs to read the update to the user, otherwise how would they know? I would recommend going to office hours for this issue. Is this a sample app bug or an issue that needs to be changed in the platform?

**Comment by Truptik12** [comment]
Verified the bug in the latest environment. The bug is fixed with Narrator, but the NVDA is still announcing the unnecessary information
[6733 Fixed Narrator.webm](https://user-images.githubusercontent.com/86710491/183897380-70576ae5-4016-45ec-a704-1321e7830ada.webm)
[6733 Not Fixed NVDA.webm](https://user-images.githubusercontent.com/86710491/183897495-23547c98-cde2-409b-938c-65c038815eb8.webm)

**Comment by merriemcgaw** [comment]
@dipeshmsft  if you use [Accessibility Insights for Windows](https://accessibilityinsights.io/) we can see the UIA hierarchy. If it is correct, and Narrator is working correctly we can resolve as external and have the tester file an issue with NVDA directly.

**Comment by dipeshmsft** [comment]
Yes, the UIA hierarchy is correct and Narrator is working fine, This is a bug in NVDA, hence closing the issue.

**Comment by merriemcgaw** [comment]
@Truptik12 can you help us file the external bug against NVDA so we can resolve this as external?

**Comment by Truptik12** [comment]
Verified the bug in the above environment. The bug does not repro. Hence closing the bug. Please find the screenshot attached

![image](https://user-images.githubusercontent.com/86710491/192529215-7bcad3ef-f4f9-4dce-a504-3a59279f8b58.png)

**Comment by Truptik12** [comment]
Verified the bug in the above environment. The bug does not repro. Hence closing the bug. Please find the screenshot attached

[6733 (2).zip](https://github.com/dotnet/wpf/files/9655982/6733.2.zip)
